restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/client/api/CallApiClient.java

```diff
@@ -0,0 +1,163 @@
+    private ActorRef requestee;
+
+    private ActorRef httpAsycClientHelper;
+
+    public CallApiClient(final Sid callSid, final DaoManager storage) {
+        super();
+        this.callSid = callSid;
+        this.storage = storage;
+    }
+    @Override
+    public void onReceive(Object message) throws Exception {
+        final Class<?> klass = message.getClass();
+        final ActorRef sender = sender();
+        ActorRef self = self();
+
+        if (logger.isInfoEnabled()) {
+            logger.info(" ********** CallApiClient " + self().path() + " Sender: " + sender);
+            logger.info(" ********** CallApiClient " + self().path() + " Processing Message: " + klass.getName());
+        }
+        if (Hangup.class.equals(klass)) {
+            onHangup((Hangup) message, self, sender);
+        } else if (DownloaderResponse.class.equals(klass)) {
+            onDownloaderResponse(message, self, sender);
+        }
+    }
+
+    private void onDownloaderResponse(Object message, ActorRef self, ActorRef sender) {
+        final DownloaderResponse response = (DownloaderResponse) message;
+        if (logger.isInfoEnabled()) {
+            logger.info("Call api response succeeded " + response.succeeded());
+            logger.info("Call api response: " + response);
+        }
+        requestee.tell(message, self);
+    }
+
+    protected void onHangup(Hangup message, ActorRef self, ActorRef sender) throws URISyntaxException, ParseException {
+        requestee = sender;
+        callDetailRecord = getCallDetailRecord();
+        if(callDetailRecord == null){
+            logger.error("could not retrieve conferenceDetailRecord");
+        } else {
+            try {
+                URI uri = new URI("/restcomm"+callDetailRecord.getUri());
+                uri = UriUtils.resolve(uri);
+                if (logger.isInfoEnabled())
+                    logger.info("call api uri is: "+uri);
+
+                Header[] headers = {
+                        new BasicHeader(HttpHeaders.AUTHORIZATION, getSuperAdminAuthenticationHeader())
+                        ,new BasicHeader("Content-type", "application/x-www-form-urlencoded")
+                        ,new BasicHeader("Accept", "application/json")
+                    };
+
+                ArrayList<NameValuePair> postParameters = new ArrayList<NameValuePair>();
+                postParameters.add(new BasicNameValuePair("Status", "Completed"));
+
+                httpAsycClientHelper = httpAsycClientHelper();
+                HttpRequestDescriptor httpRequestDescriptor =new HttpRequestDescriptor(uri, "POST", postParameters, -1, headers);
+                httpAsycClientHelper.tell(httpRequestDescriptor, self);
+            } catch (Exception e) {
+                logger.error("Exception while trying to terminate call via api: ", e);
+            }
+        }
+    }
+
+    private String getSuperAdminAuthenticationHeader(){
```

Comment:
@maria-farooq since the ConferencesEndpoint.updateConference() can be used by any account with `RestComm:Modify:Conferences` permision, why we create LCM using super admin credentials? 
I think we should use the credentials of the account that requested the update initially